src/server.py

```diff
@@ -28,7 +28,7 @@
 # declare the application and populate some details
 APP = FastAPI(
     title='Answer coalesce - A FastAPI UI/web service',
-    version='2.1.1'
+    version='2.2.0'
```

Comment:
there are 2 places to update the version. here and a little further down in construct_open_api_schema()